deps/juggler-v4/test.js

```diff
@@ -5,6 +5,12 @@
 
 'use strict';
 
+var nodeMajor = +process.versions.node.split('.')[0];
```

Comment:
Is this a temporary solution until we drop Node.js 6.x from our CI matrix in juggler and/or connectors? If so, let's put a comment to remind ourselves down the road.